### PR TITLE
Feature/partition

### PR DESCRIPTION
**Pull request by TkDodo**

### CODE REVIEW

File: src/partition.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+): [T[], T[]];
+
+/**
+ * Splits a collection into two groups, the first of which contains elements the `predicate` function matches, and the second one containing the rest.
+ * @param predicate the grouping function
+ * @returns the array of grouped elements.
+ * @signature
+ *    R.partition(fn)(array)
+ * @example
+ *    R.pipe(['one', 'two', 'forty two'], R.partition(x => x.length === 3)) // => [['one', 'two'], [['forty two']]
```
Comment (zorzysty) [inline src/partition.ts:29]:
```suggestion
 *    R.pipe(['one', 'two', 'forty two'], R.partition(x => x.length === 3)) // => [['one', 'two'], ['forty two']]
```

Comment (TkDodo) [reply src/partition.ts]: thank you. I applied the suggestions

**Review by zorzysty — APPROVED** [review]

**Review by PeterMK85 — APPROVED** [review]

**Comment by vesse** [comment]
When will this be released?

**Comment by TkDodo** [comment]
it was released right now:

https://github.com/remeda/remeda/releases/tag/v0.0.33

**Comment by vesse** [comment]
@TkDodo `index.ts` is missing the export though

**Comment by TkDodo** [comment]
damn! can you make a quick PR to contribute this please?

**Comment by Bessonov** [comment]
Just a side note: from my point of view, this function is just a special case of `groupBy`:

```ts
const { selected, unselected } = groupBy(
	items,
	item => (item.selected ? 'selected' : 'unselected'),
)
```

**Comment by TkDodo** [comment]
Indeed - but a quite common one :)

**Pull request merged**